packages/shared/src/features/onboarding/shared/Box.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import type { ReactElement } from 'react';
+import { VIcon } from '../../../components/icons/V';
+import { IconSize } from '../../../components/Icon';
+import classed from '../../../lib/classed';
+
+const Box = classed(
+  'div',
+  'flex flex-col gap-4 p-4 rounded-16 border border-border-subtlest-secondary bg-surface-float text-text-primary',
+);
+
+export interface BoxBaseProps {
+  className?: string;
+}
+
+export interface BoxListProps extends BoxBaseProps {
+  title: string;
+  items: {
+    text: string;
+  }[];
+}
+
+export const BoxList = ({
+  title,
+  items,
+  className,
+}: BoxListProps): ReactElement => {
+  return (
+    <Box className={className}>
+      <h3 className="font-bold typo-body">{title}</h3>
+      <ul className="flex flex-col gap-1">
+        {items.map((item) => (
+          <li key={item.text} className="flex flex-row items-center gap-1">
+            <VIcon size={IconSize.XSmall} />
+            <span className="typo-callout">{item.text}</span>
+          </li>
+        ))}
+      </ul>
+    </Box>
+  );
+};
+
+export interface BoxContentImageProps extends BoxBaseProps {
+  title: string;
+  content: string;
+  image: string;
+  imageAlt?: string;
```

Comment:
In Checkbox I've added it using 

```tsx
image?: ComponentProps<'img'>;
```

So we can add as many standard image attributes under this property (src/alt/...). Maybe we can use it also here in order to make it standard. Wdyt?

Reply:
Fantastic idea!